schemas/HPXMLBaseElements.xsd

```diff
@@ -5533,4 +5519,17 @@
 		</xs:sequence>
 		<xs:attribute name="dataSource" type="DataSource"/>
 	</xs:complexType>
+	<xs:complexType name="StormType">
+		<xs:sequence>
+			<xs:group ref="SystemInfo"/>
+			<xs:element minOccurs="0" name="GlazingMaterial" type="GlazingMaterial"/>
+			<xs:element minOccurs="0" name="GlassType" type="GlassType"/>
+			<xs:element minOccurs="0" name="FrameType" type="WindowFrameType"/>
+			<xs:element minOccurs="0" name="Location" type="StormLocation"/>
+			<xs:element minOccurs="0" name="Operable" type="HPXMLBoolean"/>
```

Comment:
@shorowit There's conflict to resolve with v4_0 branch where warning is added and `FractionOperable` is added. I wanted to exclude the deprecated element from `StormDoor`, but it will introduce more complexity that may not worth to do.

Reply:
I wouldn't think that Operable/FractionOperable make sense for a storm door. Isn't a door operable by definition?

Reply:
I agree, let me try splitting the type and removing `Operable`.